hypothesis-python/docs/details.rst

```diff
@@ -632,3 +632,30 @@ it creates.  For example:
     use it in any way outside of type hints.  The only supported way to
     construct objects of this type is to use the functions provided by the
     :mod:`hypothesis.strategies` module!
+
+
+.. _pytest-plugin:
+
+----------------------------
+The Hypothesis Pytest Plugin
```

Comment:
nit: the style convention in the pytest docs is to use a lowercase “p”

e.g. https://docs.pytest.org/en/latest/existingtestsuite.html